stubs/setuptools/setuptools/build_meta.pyi

```diff
@@ -1,6 +1,6 @@
-from _typeshed import Incomplete
 from typing import Any
 
+from colletions.abc import Mapping
```

Comment:
```suggestion
from collections.abc import Mapping
```

Reply:
Oops, thanks! I'm trying to test this locally against scikit-build-core with mypy ATM. "--custom-typeshed-dir" doesn't work for stubs, I think, so I'm adding via stubs dir. I'll undraft once it works! :)

Reply:
Yeah, for testing third-party stubs you have to use the `MYPYPATH` environment variable to point to the directory

Reply:
Ahh, just saw this. Okay, can keep trying with MYPYPATH...

Reply:
Ahh, just realized these dirs are nested! Sorry for the noise.